src/Mod/Draft/draftguitools/gui_edit_draft_objects.py

```diff
@@ -123,7 +132,43 @@ def update_preview_object(self, edit_command, obj, node_idx, v):
             if obj.Closed:
                 pointList.append(pointList[0])
             edit_command.ghost.updateFromPointlist(pointList)
-        
+
+    def add_point(self, edit_command, obj, pos):
+        """Add point to obj.
+        """
+        # self.setSelectState(obj, True)
+        selobjs = Gui.ActiveDocument.ActiveView.getObjectsInfo((pos[0],pos[1]))
+        if not selobjs:
+            return
+        for info in selobjs:
+            if not info:
+                return
+            if obj.Name != info["Object"]:
+                return
+            if utils.get_type(obj) == "Wire" and 'Edge' in info["Component"]:
```

Comment:
Is this check still needed? Can instance of this class (not subclasses that override this method) ever edit anything else?

Reply:
I was not sure about the check 'Edge' in info["Component"], but i tried and it works, so, perfect to remove it!

Reply:
Right, I was mostly talking about the Wire check, but I guess that a Wire can never produce anything else than edges, right?

Reply:
I think so. maybe it can produce an invalid shape (even if i never experienced that). but if so, the object is not displayed, so no way to be able to retrieve it from the scenegraph....
Uhhh, I got it. If your wire have fillets or chamfers, the editpoints are not coincident with `Draft_Wire` vertexes, so the user can `mouseover` a vertex and display context menu. In this usecase `info` does not contain edges, but vertexes! I'm reverting this change. thank you!

Reply:
Well, the correct answer was: no. A Draft Wire produces Vertexes and Edges (I do not think that in OCC you can have Edges without Vertexes). But in the Draft Edit usecase you can't mouseover a Vertex and call the `get_edit_obj_context_menu()`, because the event will be forwarded to `get_edit_point_context_menu()`. This unless you have the Vertexes not in sync with the edit points like when the wire is filleted or chamfered.